add command to link unlinked bible texts

schoenwaldnils opened the pull request.

msakowski commented:
This still doesn't work for me.

msakowski commented:
If you test these 3 examples, only the second one will be linked:

(1. Mose 1:1)
(2Mo 2:2)
- 3. Mo 3:3

schoenwaldnils commented:
@msakowski sorry this one got out of hand 😬
I updated the function and added a setting to keep or update the link texts.
Added a test for it, updated the regex and a few things here and there 🙈

dkottis commented on src/JWLibraryLinkerSettings.ts:
```diff
@@ -82,6 +82,22 @@ export class JWLibraryLinkerSettings extends PluginSettingTab {
+      .addDropdown((dropdown) =>
+        dropdown
+          .addOptions({
+            keepCurrentStructure: 'Keep current structure',
```
Could you please add translations?

schoenwaldnils replied:
Done ^^

dkottis reviewed (approved):
LGTM

msakowski commented:
It's works like a charm! Thanks much for the extra effort, it was worth it!
